storybook/shared/box.js

```diff
@@ -21,14 +21,17 @@ type BoxProps = {
 
 export default function Box({
   children,
-  margin = [0, 0.5, 1],
+  margin,
   style,
   type,
   value = type,
   ...props
 }: BoxProps) {
   return (
-    <Grid margin={margin} {...props}>
+    <Grid
+      margin={typeof margin === 'undefined' ? [0, 0.5, 1] : margin}
```

Comment:
flow still (v.0.54) chokes on mixed types with default params for function components. To work around it, instead of adding `$flowDisable` I just moved it to `margin`